### PR TITLE
Bug 1121004 - [Loop] The timer indicating the duration of the call does not seem to be correct

### PR DESCRIPTION
**Pull request by frsela**

### CODE REVIEW

File: app/js/helpers/countdown.js [hunk 1]
```diff
@@ -12,11 +12,19 @@
...
-  
+
   function _reset() {
     _counter = 0;
   }
 
+  function _paint(minutes, seconds) {
+    var len = _countdownElements.length;
+    for (var i = 0; i < len; i++) {
+      _countdownElements[i].textContent = _beautify(minutes) + ':' +
```
Comment (crdlc) [inline app/js/helpers/countdown.js]: _beautify outside of this loop

File: app/js/helpers/countdown.js [hunk 2]
```diff
@@ -24,22 +32,27 @@
       return this;
     },
     start: function(element) {
+      if (_counterTimer !== null) {
+        debug && console.log('Warning, a countdown timer is running!');
```
Comment (crdlc) [inline app/js/helpers/countdown.js]: initialize debug please with Config.debug

Comment (frsela) [inline app/js/helpers/countdown.js]: https://github.com/mozilla-b2g/firefoxos-loop-client/pull/493/files#diff-870e358d0ee3dd8219caaf698204b2a4R8

File: app/js/helpers/countdown.js (hunk 1 above)
Comment (frsela) [inline app/js/helpers/countdown.js]: +1

File: app/js/helpers/countdown.js [hunk 3]
```diff
@@ -24,22 +35,27 @@
...
+        _counter = (performance.now() - start) / 1000;
+        var minutes = Math.floor(_counter/60);
+        var seconds = Math.floor(_counter%60);
+        _paint(minutes, seconds);
       }, 1000);
     },
     stop: function() {
       clearInterval(_counterTimer);
+      _counterTimer = null;
+      _paint(0, 0);
```
Comment (crdlc) [inline app/js/helpers/countdown.js]: stop is not reset so i think that this _paint is not needed

File: app/js/helpers/countdown.js [hunk 4]
```diff
@@ -24,22 +35,27 @@
       return this;
     },
     start: function(element) {
+      if (_counterTimer !== null) {
+        debug && console.log('Warning, a countdown timer is running!');
+        return;
+      }
+      var start = performance.now();
```
Comment (crdlc) [inline app/js/helpers/countdown.js]: Wrong behavior:

Start:
00:01 - 00:02 - 00:03
Stop:
00:00
Start:
00:00 - 00:01 - 00:02

Comment (crdlc) [inline app/js/helpers/countdown.js]: Wrong behavior:

Start:
00:01 - 00:02 - 00:03
Reset:
00:00 - 00:03 - 00:04

**Pull request merged**